scripts/deployment/build-and-deploy.sh

```
@@ -0,0 +1,700 @@
+        phase_end error "Missing LOCKBOX_MASTER_KEY in $ENV_FILE"
+        exit 1
+    fi
+    if ! [[ "$LOCKBOX" =~ ^[0-9a-fA-F]{64}$ ]]; then
+        phase_end error "LOCKBOX_MASTER_KEY must be 64 hex chars"
+        exit 1
+    fi
+    log_step "Env vars OK"
+fi
+
+# Check port
+if lsof -Pi :$PORT -sTCP:LISTEN -t >/dev/null 2>&1; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)